File: src/main/java/pl/yalgrin/playnite/simplesync/dto/filter/StringItemPropertiesDTO.java
package pl.yalgrin.playnite.simplesync.dto.filter;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

import java.io.Serial;
import java.io.Serializable;
import java.util.List;

@EqualsAndHashCode
@Data
@NoArgsConstructor
@SuperBuilder
public class StringItemPropertiesDTO implements Serializable {
    @Serial
    private static final long serialVersionUID = 1808659491232511772L;

    @JsonProperty("Values")
    private List<String> values;
}

File: src/main/kotlin/pl/yalgrin/playnite/simplesync/library/dto/filter/StringItemPropertiesDTO.kt
package pl.yalgrin.playnite.simplesync.library.dto.filter

import java.io.Serial
import java.io.Serializable

data class StringItemPropertiesDTO(
    var values: List<String> = emptyList()
) : Serializable {
    companion object {
        @Serial
        private const val serialVersionUID = 1808659491232511772L
    }
}